packages/react/bundle-size/ActivityItem.fixture.js

```diff
@@ -0,0 +1,7 @@
+import * as p from '@fluentui/react/lib/ActivityItem';
```

Comment:
Out of curiosity, why is the import structured like this and not like `import { ActivityItem } from "@fluentui/react"`?

Reply:
from what I remember from various convos from the past, is because of tree shaking, v8 encouraged nested/private apis ( prior to node export maps support across the ecosystem ).

now regarding those fixtures, I generated them from the old SizeAuditor thing, where they were generated dynamically on every measurement run.